packages/@sanity/core/src/actions/graphql/listApisAction.js

```diff
@@ -0,0 +1,34 @@
+module.exports = async function listApisAction(args, context) {
+  const {apiClient, output, chalk} = context
+
+  const client = apiClient({
+    requireUser: true,
+    requireProject: true
+  })
+
+  let endpoints
+  try {
+    endpoints = await client.request({
+      url: `/apis/graphql`,
+      method: 'GET'
+    })
+  } catch (err) {
+    if (err.statusCode === 404) {
```

Comment:
I don't think this should be a 404 from the backend unless it means the route/project doesn't exist

Reply:
Yep. Let's discuss that in the backend PR.

Reply:
I'll just remove that check for now.